services.py

```diff
@@ -4,62 +4,73 @@
 
 EON = os.path.isfile('/EON')
 
+RESERVED_PORTS = {8022, 22}  # sshd
+STARTING_PORT = 8001
+
+
+def new_port(port: int):
+  port += STARTING_PORT
+  return port + len(services) if port in RESERVED_PORTS else port
```

Comment:
Let's not make it too complicated. We only need to reserve 8022. So this can just be
```python
RESERVED_PORT = 8022
STARTING_PORT = 8001

def new_port(idx: int):
  port = STARTING_PORT + idx
  return port + 1 if port >= RESERVED_PORT else port
```

Reply:
Okay gotcha. I just wanted to make sure it would be easily extensible if we needed many

Reply:
If the general solution is more complex/hard to read it's not really worth it. If we need more ports we can always change the implementation later.

Reply:
I think there's a bug with mypy, I saw a few similar issues with unpacking a tuple for a function. I'll try adding a comment to ignore the line